### PR TITLE
Migrate gi-frontend-web to gi-frontend on vite with new char card

### PR DESCRIPTION
**Pull request by nguyentvan7**
image

### CODE REVIEW

**Comment by frzyc** [comment]
Should also remove `gi-frontend-web`

File: libs/pando/tsconfig.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "ES2022",
```
Comment (nguyentvan7) [inline libs/pando/tsconfig.json:4]: @lantua is this bad

Comment (lantua) [reply libs/pando/tsconfig.json:4]: if it compiles (and test succeed), should be fine

**Review by frzyc — APPROVED** [review]
Good enough, 🚀

**Pull request merged**